### ISSUE TITLE
[bug] chained unary operators are not pushed correctly to stack 

### ISSUE THREAD

**Issue by lofcz** [codingseb/ExpressionEvaluator#80]
Consider these examples:

```csharp
int a = 0;
a = -~a;
return a; // returns 1 in c#, EE crashes
```

```csharp
int a = 0;
a = +-+-+-+-+a;
return a; // returns 0 in c#, EE crashes
```

```csharp
int a = 0;
a = a >> +-+-+-+2 << +-+-+-+-2 >> +-+-+-+-+2 << +-+-+-+-+2;
return a; // returns 0 in c#, EE crashes
```

**Comment by codingseb** [comment]
OK, I think I never used multiple unary operators until today 🤔.
I will look how to correct that 👍🏻.

**Comment by lofcz** [comment]
> OK, I think I never used multiple unary operators until today 🤔.
> I will look how to correct that 👍🏻.

I actually use the first one occasionally to flex on my fellow peers 😄

**Comment by lofcz** [comment]
@codingseb great job with the new features. Should you have some spare time, would you please look into this issue? Thank you!

**Comment by codingseb** [comment]
OK I think I am close of the solution. Just need to make one or two more tests and I will publish it. It is for very soon.

**Comment by lofcz** [comment]
Great job sorting this out! I've tested chained unary operators branch and found no issues.

**Comment by codingseb** [comment]
> Great job sorting this out! I've tested chained unary operators branch and found no issues.

@lofcz yes all your examples were already working. I just made it work for custom left operands only operators.

I just published version 1.4.25.0 that manage this issue. Thanks again for all @lofcz

**Comment by lofcz** [comment]
@codingseb you are the best!
